docs/index.rst

```diff
@@ -42,9 +42,7 @@ Developing for Intranet
 Using Intranet as a Developer
 -----------------------------
 - `API Root <https://ion.tjhsst.edu/api/>`_
-- `API Demo <https://www.tjhsst.edu/~2016jwoglom/ion-api-demo.html>`_
```

Comment:
I'm guessing you removed these links because the Director 4.0 upgrade broke James's site. I've managed to get that site working again, so it's probably OK to leave it in place.